docker/hooks/build

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+
+docker image build --build-arg SOURCE_BRANCH=$SOURCE_BRANCH \
+	--label "org.opencontainers.image.created"=$(date +%Y-%m-%d) \
+	--label "org.opencontainers.image.revision"=$SOURCE_COMMIT \
+	--label "org.opencontainers.image.version"=$DOCKER_TAG \
+	-t $IMAGE_NAME \
```

Comment:
Based in a recent experience solving a build problem in release/2.4.0 branch (see PR https://github.com/telefonicaid/fiware-orion/pull/3702), we suggest the following:

```suggestion
# -e, to end execution immediately if 'cd ..' fails
# -x, to print output as we go
set -ex

# due to Dockerfile is not in the repository root, but in a subdirectory and we use -f docker/Dockerfile
cd ..

docker image build --build-arg GIT_REV_ORION=$SOURCE_BRANCH \
	--label "org.opencontainers.image.created"=$(date +%Y-%m-%d) \
	--label "org.opencontainers.image.revision"=$SOURCE_COMMIT \
	--label "org.opencontainers.image.version"=$DOCKER_TAG \
	-f docker/Dockerfile \
	-t $IMAGE_NAME \
```

CC: @IvanHdzC @AlvaroVega

Reply:
@jason-fox if you prefer, we can do it the other way around: backport PR https://github.com/telefonicaid/fiware-orion/pull/3702 to master, then adapt your PR to the new file in master.

What do you think?